.ci/schedule-packages.sh

```diff
@@ -38,4 +38,7 @@ for i in "${!PACKAGES[@]}"; do
 done
 
 # Write the parameters to a file .ci/schedule-params.txt
-declare -p PARAMS > .ci/schedule-params.txt
+declare -p PARAMS >.ci/schedule-params.txt
+
+# Debug: show params
+cat .ci/schedule-params.txt
```

Comment:
https://github.com/chaotic-cx/chaotic-poc/blob/main/.github/workflows/on-commit.yml#L55

Something like in here might be smarter?